app/README.md

```diff
@@ -8,3 +8,21 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 
 A GUI client for OpenMLS
 
+## Code organization
+
+The app code is organized in the following way:
+
+- Each feature is a separate module in the `lib` directory.
+- Each module in the `lib` exports all its public APIs in the dart file with the
+same name as the module. Other modules should import this dart file.
+- Reusable widgets are in the `widgets` module.
+- All styles and theme related code in the `theme` module.
+- The module `core` contains the Rust generated code and extensions.
+
+### Naming conventions
+
+- `View` is a pure widget without any providers.
+- `Container` wraps a `View` and creates all required providers.
+- `ScreenView` is a root widget in the navigation without any providers.
+- `Screen` wraps a `ScreenView` and creates all required providers. This is the
+  actual class used in the navigation stack.
```

Comment:
This is nice. I think it would also be good to add "pane" since it's being used for large screens.

Reply:
Done in [06d716d](https://github.com/phnx-im/infra/pull/298/commits/06d716d77d89b894fb4d98d6b5fe7486292514c6)